addons/website/static/src/scss/website.scss

```diff
@@ -1989,25 +2001,39 @@ header {
             } @else if o-website-value('header-links-style') == 'border-bottom' {
                 // There is no way to control navbar links vertical padding in BS4
                 // independently from nav ones, just double them here instead
-                padding-top: ($nav-link-padding-y * 2);
-                padding-bottom: ($nav-link-padding-y * 2);
+                padding-top: $nav-link-padding-y * 2;
+                padding-bottom: $nav-link-padding-y * 2;
                 border-bottom: $nav-link-padding-y solid transparent;
-
                 // Replace horizontal paddings by margins (do this with an extra
                 // class to override .navbar-expand-* paddings priority).
                 .navbar & {
                     padding-left: 0;
                     padding-right: 0;
                     margin: 0 $navbar-nav-link-padding-x;
                 }
+            } @else if o-website-value('header-links-style') == 'underline' {
+                & > span {
+                    border-bottom: 0.125em solid transparent;
+                }
             }
         }
     }
 
-    @if index(('outline', 'border-bottom'), o-website-value('header-links-style')) {
+    @if index(('outline', 'border-bottom', 'underline', 'bold'), o-website-value('header-links-style')) {
+        $header-text-color: o-website-value('header-text-color');
         .nav-link.active,
         .show > .nav-link {
-            border-color: currentColor;
+            @if index(('outline', 'border-bottom'), o-website-value('header-links-style')) {
+                @include active-link-border-color($header-text-color);
+            }
+            @if o-website-value('header-links-style') == 'underline' {
+                & > span{
+                    @include active-link-border-color($header-text-color);
+                }        
+            }
+            @if o-website-value('header-links-style') == 'bold' {
+                font-weight: bold !important;
```

Comment:
Do we want this on mobile, unlike the other option styles?
Also, why are we applying `border-color` on mobile if `border-bottom` are not applied there?